pkgs/development/python-modules/anitopy/default.nix

```diff
@@ -0,0 +1,18 @@
+{ buildPythonPackage, fetchFromGitHub, lib }:
+buildPythonPackage rec {
+  pname = "anitopy";
+  version = "2.1.1";
+  src = fetchFromGitHub {
+    owner = "igorcmoura";
+    repo = "${pname}";
+    rev = "v${version}";
+    hash = "sha256-xXEf7AJKg7grDmkKfFuC4Fk6QYFJtezClyfA3vq8TfQ=";
+  };
```

Comment:
```suggestion

  src = fetchFromGitHub {
    owner = "igorcmoura";
    repo = "${pname}";
    rev = "v${version}";
    hash = "sha256-xXEf7AJKg7grDmkKfFuC4Fk6QYFJtezClyfA3vq8TfQ=";
  };
```

Reply:
You are also missing the format attribute

Reply:
I'm sorry but what exactly is different in this?

Reply:
There is just a space more. just a formating change.

Reply:
and with the format attribute I mean something like `format = "setuptools";`